website_sale_with_tax/views/product_cart_products_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+<data>
+
+    <template id="products_cart_with_tax" inherit_id="website_sale.cart">
+        <xpath expr="//td[@name='price']/span" position="attributes">
+            <attribute name="t-field">line.product_id.price_with_tax</attribute>
```

Comment:
Better to make invisible this field and add another one to avoid inherit errors from other modules that modifies this one